hamilton/plugins/jupyter_magic.py

```diff
@@ -260,6 +248,157 @@ def insert_module(self, line):
         # insert source code as text in the next cell
         self.shell.set_next_input(header + module_source, replace=False)
 
+    @magic_arguments()  # needed on top to enable parsing
+    @argument(
```

Comment:
Better to use a deafult value here? See docs: https://ipython.readthedocs.io/en/stable/api/generated/IPython.core.magic_arguments.html.

Reply:
we'll refactor to match jupyter v2 once we're there